app/frontend/javascript/custom_google_analytics_4_events.js

```diff
@@ -0,0 +1,23 @@
+// Send a custom Google Analytics event with data-attributes on a link.
+// Details about migrating to the GA4 event tracker are here:
+// https://developers.google.com/analytics/devguides/migration/ua/analyticsjs-to-gtagjs#measure_events_with_the_default_tracker
+// Note that app/frontend/javascript/custom_google_analytics_universal_events.js
+// contains a method that is trigged by the same click.
+$(document).on('click', '*[data-analytics-category]', function(e) {
+  if (typeof gtag === 'function') {
```

Comment:
I'd just add a comment that this is checkking to make sure `gtag` is defined, which it should be only if GA4 is enabled. And same on universal GA.  at first I missed this subtlety and wasn't sure we were properly guarding.

Reply:
Done!